abi-bindings/go/governance/ValidatorSetSig/packing.go

```diff
@@ -29,37 +29,29 @@ func init() {
 	}
 }
 
-func PackValidatorSetSigWarpPayload(message ValidatorSetSigMessage) ([]byte, error) {
+func (m *ValidatorSetSigMessage) Pack() ([]byte, error) {
 	args := abi.Arguments{
 		{
 			Name: "validatorSetSigMessage",
 			Type: validatorSetSigMessageType,
 		},
 	}
-	return args.Pack(message)
+	return args.Pack(m)
 }
 
-func UnpackValidatorSetSigWarpPayload(messageBytes []byte) (ValidatorSetSigMessage, error) {
+func (m *ValidatorSetSigMessage) Unpack(messageBytes []byte) error {
 	args := abi.Arguments{
 		{
 			Name: "validatorSetSigMessage",
 			Type: validatorSetSigMessageType,
 		},
 	}
 	unpacked, err := args.Unpack(messageBytes)
-	fmt.Println("unpacked: ", unpacked)
 	if err != nil {
-		return ValidatorSetSigMessage{}, fmt.Errorf("failed to unpack to ValidatorSetSigMessage with err: %v", err)
+		return fmt.Errorf("failed to unpack to ValidatorSetSigMessage with err: %v", err)
 	}
-	type validatorSetSigWarpPayload struct {
-		ValidatorSetSigMessage ValidatorSetSigMessage `json:"validatorSetSigMessage"`
-	}
-	var payload validatorSetSigWarpPayload
-	err = args.Copy(&payload, unpacked)
-	if err != nil {
-		return ValidatorSetSigMessage{}, err
-	}
-	return payload.ValidatorSetSigMessage, nil
+	err = args.Copy(&m, unpacked)
```

Comment:
Is the `Copy` here necessary, or could we `Unpack` directly into `&m`? If it is necessary, lets just `return args.Copy(...)`.